content/webapp/pages/concepts/[conceptId].tsx

```diff
@@ -149,21 +149,27 @@ type ImagesTabPanelProps = {
   id: string;
   link: LinkProps;
   results: ReturnedResults<ImageType>;
+  totalResults: number | undefined;
 };
 const ImagesTabPanel: FunctionComponent<ImagesTabPanelProps> = ({
   id,
   link,
   results,
+  totalResults,
 }) => {
+  const showSeeMore = totalResults !== undefined && totalResults > 0;
```

Comment:
It's not clear why I needed to do this. 

```
{totalResults && (
          <SeeMoreButton
            text="All images"
            totalResults={totalResults}
            link={link}
          />
        )}
```

Would display "0" in place of the component which was a surprise as 0 is falsy here ... perhaps it's getting evaluated as a string here?

Reply:
The value of the first falsy operand gets returned (in this case the number 0) and React will render anything that is type of number (or string). You could do Boolean(totalResults) instead.

Reply:
or !!totalResults

Reply:
still think false gets returned but React doesn't do anything with it